tests/integration/specs/multiple-resource-links-test.js

```diff
@@ -0,0 +1,91 @@
+var get = Ember.get, set = Ember.set;
+var env;
+var responses, fakeServer;
+
+module('integration/specs/multiple-resource-links-test', {
+    setup: function() {
+        fakeServer = stubServer();
+
+        responses = {
+            posts_not_compound: {
+                data: [{
+                    type: 'posts',
+                    id: '1',
+                    title: 'Rails is Omakase',
+                    links: {
+                        author: {
+                            resource: '/authors/2',
+                            type: 'authors',
+                            id: '2'
+                        }
+                    }
+                }, {
+                    type: 'posts',
+                    id: '2',
+                    title: 'TDD Is Dead lol',
+                    links: {
+                        author: {
+                            resource: '/authors/1',
```

Comment:
You are correct. The wording around this has been updated. The original wording was softer and it could be the link to the resource. The new wording still does not say **MUST**, but the inference is that the URL will return the current related resource. I feel that should be the responsibility of `self`, but I will update accordingly.